content.md

```diff
@@ -77,6 +77,8 @@ When your server hits the debugger line, it will pause, and you'll enter the deb
 - `continue` or `c`: Resume normal execution until the next breakpoint.
 - `vars`: Display local variables and their values.
 - `p <expression>`: Evaluate and print any Ruby expression.
+  
+<aside>In debugging, the term "breakpoint" refers to the specific point in the code that causes the program to stop executing until it receives further input.</aside>
```

Comment:
FYI, this broke the lesson page. It's a bug on our side, but just wanted to note I just fixed it on this commit: https://github.com/appdev-lessons/debugging/commit/7c788402be73aa6b4f76cd41ac9cb3177ddd6044

by changing this to:

```
<aside>
Aside text here.
</aside>
```

Reply:
Concerning that no students brought it up until I just got a feedback note about it today!